### ISSUE TITLE
The name...

### ISSUE THREAD

**Issue by rogeralsing** [clusterinc/skit#4]
I'm sure you already know, but "Skit" means "Shit" in Swedish.
No matter how good the framework is, you would have a very hard time convincing your boss/project manager to go for this lib.. just saying..

**Comment by taylorhughes** [comment]
thanks for reporting — going to leave it open in case any other bugs get filed, but I am not going to rename everything just yet. :)

**Comment by wub** [comment]
> No matter how good the framework is, you would have a very hard time convincing your boss/project manager to go for this lib

You mean:

> No matter how good the framework is, you would have a very hard time convincing your **Swedish** boss/project manager to go for this lib

Luckily, mine isn't Swedish :grimacing:

**Comment by ghost** [comment]
I actually though it was an April Fools joke or something similar to DICSS. It felt to good to be true sometimes when reading the homepage. I had a hard time to not laugh :)

Maybe add a pronounce? I assume non-swedish users pronounce it "Ess-Kit"?

**Comment by IamAdamJowett** [comment]
Considering how I feel when it is time to setup another monolithic framework for a new job, the name is completely appropriate. Great work mate.

**Comment by nstadigs** [comment]
At least you get publicity. This framework seems to be spreading like wildfire on swedish chatrooms/forums.

**Comment by zond** [comment]
This is wonderful, I always felt that JavaScript was crap, now I can use a framework that acknowledges this!

**Comment by lodnert** [comment]
Haha, good name :)

**Comment by pariz** [comment]
I disagree with @madr. Skit feels warm, soft and natural. It is clear that the name has gone through a rigorous mental digestive system.

**Comment by taylorhughes** [comment]
:100: :hankey: :+1:

**Comment by mblz** [comment]
Seems like a good name to me, rolls off the tongue. Skit'r done.

**Comment by bmull** [comment]
"Poop" is actually also "poop" in Swedish, however "feces" is "avföring". #TheMoreYouKnow

**Comment by pakastin** [comment]
I also thought this was a joke: "What is skit?", "What's skit made of?", "See skit in action" :D.
I'm from Finland, Sweden is our second official language.

Good to know this is for real. Yet I'm still not 100% sure if I'm getting fooled :)

**Comment by croon** [comment]
@bmull Seeing as it's a highly relevant topic, I have to clarify that "poop" is actually "bajs" in Swedish.

**Comment by jonahx** [comment]
This really ought to be closed with a `feature, not a bug` tag.

**Comment by taylorhughes** [comment]
@jonahx if I do that, it tends to pop up again when someone else reports it. any way to prevent that?

**Comment by jonahx** [comment]
Oh I was just making a joke.  Not sure what the real answer to your question is, but keeping it open seems fine.

**Comment by taylorhughes** [comment]
@jonahx oh, I ruined your joke. poop.

**Comment by masters3d** [comment]
![image](https://cloud.githubusercontent.com/assets/6539412/8601693/871febc0-2622-11e5-891e-4f4a045d4827.png)

**Comment by pakastin** [comment]
https://translate.google.com/#auto/en/skit

**Comment by taylorhughes** [comment]
@pakastin did you mean http://translate.google.com/#sv/en/skit ? "auto" defaults to english for me

**Comment by pakastin** [comment]
Oh, I didn't realize it varies.. :)

**Comment by jenstornell** [comment]
"skit" could be translated to "shit", but when I read "clusterinc/skit", it for me translates more into "crap", kind of like junk. Like a temporary repository for unwanted crap/junk.

But keep the name, it draws attention.

>[SEE SKIT IN ACTION](https://skitjs.com/)

No thanks! :smile:

/A swedish guy
